cypress/fixtures/userRegistrationResponse.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": 78,
+  "provider": "email",
+  "uid": "user@email.com",
+  "allow_password_change": false,
+  "email": "user@email.com",
+  "created_at": "2021-10-01T13:57:29.960Z",
+  "updated_at": "2021-10-01T13:57:29.968Z"
+}
```

Comment:
This fixture is still not what the server will respond with

Reply:
Noticed and done!